packages/miniflare/src/workers/assets/rpc-proxy.worker.ts

```diff
@@ -0,0 +1,33 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class SpecialDefault extends WorkerEntrypoint {
+	fetch(request) {
+		return this.env.ROUTER_WORKER.fetch(request);
+	}
+
+	constructor(...args) {
+		return new Proxy(super(...args), {
```

Comment:
Wow did not know you could return from a constructor. That's a neat trick that certainly makes proxying cleaner.